src/backend/marsha/core/api.py

```diff
@@ -131,7 +131,7 @@ def initate_upload(self, request, pk=None):
         )
 
         # Reset the upload state of the video
-        Video.objects.update(upload_state=PENDING)
```

Comment:
Because it issues a query to all records.
The normal way to do it would be:
```
video.upload_state = PENDING
video.save()
```
I'm calling the query directly to avoid all side effects and make sure I just reset the field. Not sure it was a good idea.